### DISCUSSION TITLE
Pi-hole login

### DISCUSSION THREAD

**Discussion by lightsout** [tteck/Proxmox#1419] [Help Needed] [answered]
Sorry if I am missing the obvious, but I don't know what the login is for pihole. It installed fine, it asked me for a root password which I input.

I can not SSH as I do not have a user name, and I cannot login to admin panel as it is not accepting the password I made.

**Comment by tteck** [comment, marked as answer]
SSH username is `root`
![Screenshot 2023-05-12 6 28 23 PM](https://github.com/tteck/Proxmox/assets/21342670/0761477a-6de6-47f9-9c52-839e40a88ac8)

**Reply by lightsout** [reply]
> SSH username is `root` ![Screenshot 2023-05-12 6 28 23 PM](https://user-images.githubusercontent.com/21342670/238092115-0761477a-6de6-47f9-9c52-839e40a88ac8.png)

Oh man what kind of noob doesn't try "root". Thank you and sorry for wasting your time. I saw the notes there on the page but couldn't get passed the login.

**Reply by Miq1** [reply]
Sorry to pull up this old discussion, but I have the very same problem, I in fact used ``root``, but for the life of me cannot guess the password.

**Reply by tteck** [reply]
Did you set a password during creation using Advanced settings?

**Reply by Miq1** [reply]
I did, but it did not work. I found a solution in the meantime: by ``pvm enter <id>`` I was able to use ``passwd`` to again enter a password. After that it worked.

**Comment by pakair** [comment]
It did not work, I went thru the advanced settings, created a root password. CT created. Attempted to log on to 10.0.0.40/admin, entered my pw ... Wrong password. According to the text below, After installing Pi-hole for the first time, a password is generated and displayed to the user. Where? Current pi-hole version is v5.18.3

Here's the step missing ... after the CT is up and running (after reboot), click on Shell to get to that CT. The first time, it logs on as root without password. Then, issue the command, pihole -a -p to set a password. Then, I can log on to the Web interface. Maybe you can make the instructions clearer.
